numpy/ma/core.py

```diff
@@ -4793,7 +4794,8 @@ def reshape(self, *s, **kwargs):
         Notes
         -----
         The reshaping operation cannot guarantee that a copy will not be made,
```

Comment:
Could we rewrite a bit more, like
```
By default, the reshaping operation will make a copy if a view with different strides
is not possible. To ensure a view, pass ``copy=False``.
```
I don't think you need to mention setting a shape in place at all.